docusaurus/docs-internals/brizy-editor/introduction.md

```diff
@@ -3,3 +3,35 @@ sidebar_position: 1
 ---
 
 # Introduction
+
+The environment where users design pages and oversee site settings is referred to as the `editor`. This environment comprises two key segments: the `preview` and the `editor`. Within each segment, there are additional areas that third-party developers can expand.
+
```

Comment:
i think this text is not related, because in your introduction you need to write what user will read at the `Internals` module